app/utils/github.ts

```diff
@@ -0,0 +1,19 @@
+import { Octokit } from 'octokit';
+import { createAppAuth } from '@octokit/auth-app';
+import getConfig from 'next/config';
+
+const { serverRuntimeConfig = {} } = getConfig() || {};
+const { gh_app_id, gh_app_installation_id, gh_app_private_key } = serverRuntimeConfig;
+
+export const authenticateToGithub = async () => {
+  const octokit = new Octokit({
```

Comment:
Just wondering if we should reuse this instance instead of instantiating on each hit, e.g

``` js
let cachedOctokit = null;

export const authenticateToGithub = async () => {
  if (cachedOctokit) return cachedOctokit;
  
  cachedOctokit = new Octokit({...})
 ```